website/src/components/Messages/MessageTableEntry.tsx

```diff
@@ -143,7 +153,7 @@ export function MessageTableEntry({
                   emoji={{ name: emoji, count }}
                   checked={emojiState.user_emojis.includes(emoji)}
                   userReacted={emojiState.user_emojis.length > 0}
-                  userIsAuthor={message.user_is_author}
+                  userIsAuthor={!!message.user_is_author}
```

Comment:
do we need this double negation?

Reply:
Just to make TS happy cause backend mark it as optional (nullable)